ui/raidboss/data/05-shb/trial/innocence.js

```diff
@@ -80,24 +70,7 @@
       regexJa: Regexes.startsUsing({ id: '3EA3', source: 'イノセンス' }),
       regexCn: Regexes.startsUsing({ id: '3EA3', source: '无瑕灵君' }),
       regexKo: Regexes.startsUsing({ id: '3EA3', source: '이노센스' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            ja: '自分にタンクバスター',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            ja: data.ShortName(matches.target) + 'にタンクバスター',
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
losing the condition here

can you go back and check all your tankbusters and make sure that you haven't dropped conditions elsewhere? I'm sure I missed some as well

Reply:
Same as before, here.